bitnami/external-dns/templates/serviceaccount.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.rbac.create }}
+{{- if .Values.rbac.serviceAccount.create }}
```

You need to check both conditions, something like:
```suggestion
{{- if and .Values.rbac.create .Values.rbac.serviceAccount.create }}
```